framework/src/utils/BicubicInterpolation.C

```diff
@@ -75,7 +75,7 @@ BicubicInterpolation::sampleDerivative(Real x1, Real x2, unsigned int deriv_var)
 
     Real d = _x1[x1u] - _x1[x1l];
 
-    if (!MooseUtils::absoluteFuzzyEqual(d, 0.0))
+    if (!MooseUtils::absoluteFuzzyEqual(d, Real(0.0)))
```

Comment:
`Real(0.0)` id probably necessary as ADReal can be constructed by a floating point number as well. But this is clunky.

Reply:
It looks like this cast may be unnecessary after all...